### ISSUE TITLE
Gitlab Create a release on GitLab 404 not found

### ISSUE THREAD

**Issue by jedmarcnocum** [webpro/release-it#456]
Good day! I'm having trouble with creating releases in Gitlab. I was getting a `connect ECONNREFUSED 127.0.0.1:443` error if origin is not provided in gitlab option as shown below 

package.json 

```
  "release-it": {
    "github": {
      "release": false
    },
    "gitlab": {
      "release": true
    },
    "npm": {
      "publish": false
    },
    "git": {
      "requireUpstream": false
    }
  }
```
the release output results in http://localhost/project_name/releases

On the other hand, passing the origin value to the option throws a `ERROR Response code 404 (Not Found)` (maybe the URL is incorrect? please let me know how to structure it)

package.json

```
  "release-it": {
    "github": {
      "release": false
    },
    "gitlab": {
      "release": true,
      "origin": "https://gitlab.com/group_name/sub_group/project_name"
    },
    "npm": {
      "publish": false
    },
    "git": {
      "requireUpstream": false
    }
  }
```

When running the release-it on --dry-run, I can visit the output URL 
"https://gitlab.com/group_name/sub_group/project_name/releases"
which redirects me to the release page.

I already have a personal access token with API enabled as the documentation suggested. 

The repository that I've mentioned above is a private one (Which I think what causes the error?)

**Comment by webpro** [comment]
Hi @jedmarcnocum!

Just verified that GitLaB Releases work with release-it v10.0.3.

1. In short: you should not need the `origin` option. Long: I only added `origin` to be able to use `http://localhost` (non-SSL) when GitLab didn't have Releases available yet on gitlab.com, and I could test with a local docker version of Gitlab).
2. Using a private repo should not be an issue, I just retested that as well. So no changes needed wrt public or private repos.
3. I see "group_name/sub_group/project_name" in the git url. This might be the culprit, will investigate. Are you sure "sub_group" is valid in the Git remote url?

**Comment by webpro** [comment]
The issue should be fixed in [v10.0.4](https://github.com/webpro/release-it/releases/tag/10.0.4). Had to replace a dependency that didn't handle this properly.

Just to be sure: no need to provide the `gitlab.origin` setting.

**Comment by jedmarcnocum** [comment]
Thanks @webpro ! I'll give it a try
